### PR TITLE
Gelato VRF guide added

### PR DESCRIPTION
**Pull request by leirbag95**
Here you'll find an example of how to install and use Gelato VRF on Berachain 

path: `apps/gelato-vrf`

### CODE REVIEW

File: apps/gelato-vrf/README.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# How To #10 on Gelato: VRF Jump Start
```
Comment (larrythecucumber321) [inline apps/gelato-vrf/README.md:1]: Is #10 meant to be something else?

File: apps/gelato-vrf/README.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+This repository includes contracts and configurations necessary for implementing VRF functionality on Gelato, making it easy for developers to integrate random number generation into their projects.
+
+## Contracts
+
+There are two types of VRF contracts available in this repository:
+
+1. Gelato VRF
+
+> [!NOTE]  
+> Check the deployed addresses for each network in the deployments folder.
```
Comment (larrythecucumber321) [inline apps/gelato-vrf/README.md:21]: Where is the deployments folder? If this is created after performing a certain step, then this should specify.

File: apps/gelato-vrf/README.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+- **Description**: Contract for using Gelato VRF directly.
+- **Contract Address on Berachain**: [0x2f32A4796bc8A37229FCfe8018f1fd6FBA19471C](https://bartio.beratrail.io/address/0x2f32A4796bc8A37229FCfe8018f1fd6FBA19471C)
+- **Features**:
+  - Requests randomness from Gelato VRF.
+  - Handles the fulfillment of randomness.
+
+# Create a Gelato VRF Task
+
+To create a Gelato VRF Task, please refer to the documentation [here](https://docs.gelato.network/web3-services/vrf/quick-start/deploying-your-vrf-instance).
```
Comment (larrythecucumber321) [inline apps/gelato-vrf/README.md:33]: This README should contain the contents of that link and/or instructions on how to run the contracts/scripts in the repo

File: apps/gelato-vrf/typechain-types/GelatoVRFConsumerBase.ts [hunk 4]
```diff
@@ -0,0 +1,163 @@
+/* Autogenerated file. Do not edit manually. */
```
Comment (larrythecucumber321) [inline apps/gelato-vrf/typechain-types/GelatoVRFConsumerBase.ts:1]: Typechain files are typically gitignored and included as part of the build process

Comment (leirbag95) [reply apps/gelato-vrf/typechain-types/GelatoVRFConsumerBase.ts]: Indeed, it is created with each new deployment, which is why the folder has been deleted.

File: apps/gelato-vrf/README.md (hunk 2 above)
Comment (leirbag95) [reply apps/gelato-vrf/README.md]: I've clarified in the README that the deployments folder is automatically generated at the root of the gelato-vrf/ directory after each deployment.

**Pull request merged**